src/main/kotlin/no/nav/amt/deltaker/bff/deltaker/model/Deltaker.kt

```diff
@@ -26,4 +27,13 @@ data class Deltaker(
     fun harSluttet(): Boolean {
         return status.type in AVSLUTTENDE_STATUSER || sluttdato?.isBefore(LocalDate.now().minusWeeks(2)) == true
     }
+
+    data class Samtykke(
```

Comment:
Det blir litt mange samtykker her nå 😅  men jeg vet ikke hva jeg kan kalle dette for noe...